src/System.Memory/tests/ParsersAndFormatters/Parser/ValidateParser.cs

```diff
@@ -63,9 +63,11 @@ private static bool IsParsedValueEqual<T>(T expected, T actual)
         {
             if (typeof(T) == typeof(decimal))
             {
-                int[] expectedBits = decimal.GetBits((decimal)(object)expected);
-                int[] actualBits = decimal.GetBits((decimal)(object)actual);
-                if (!expectedBits.SequenceEqual(actualBits))
+                MutableDecimal expectedBits = ((decimal)(object)expected).ToMutableDecimal();
```

Comment:
Do we not have an existing equality method on that will properly compare `+0` and `-0`?

This seems like the kind of thing that would be useful to extract into some sort of helper method